server/src/main/java/org/apache/druid/segment/metadata/AbstractSegmentMetadataCache.java

```diff
@@ -385,55 +378,26 @@ private void startCacheExec()
     );
   }
 
+  private void setInitialized(Stopwatch stopwatch)
```

Comment:
The method name is misleading here. One would assume that it would have something to do with stopWatch but it does entirely different things. 

Could it be renamed to something else?

Reply:
Renamed to `setInitializedAndReportInitTime`.